packages/nc-gui-v2/components/smartsheet-toolbar/ColumnFilter.vue

```diff
@@ -132,13 +132,12 @@ defineExpose({
               <a-select
                 v-model:value="filter.logical_op"
                 :dropdown-match-select-width="false"
-                size="small"
-                class="flex-shrink-1 flex-grow-0 elevation-0 caption !text-xs"
+                class="flex-shrink-1 flex-grow-0 elevation-0 caption"
```

Comment:
can you possibly replace the vuetify styles here so it doesn't break when we finally remove it? :)